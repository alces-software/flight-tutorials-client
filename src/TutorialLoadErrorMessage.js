// @flow
/*=============================================================================
 * Copyright (C) 2017 Stephen F. Norledge and Alces Flight Ltd.
 *
 * This file is part of Flight Tutorials.
 *
 * All rights reserved, see LICENSE.txt.
 *===========================================================================*/
import React from 'react';

const TutorialLoadErrorMessage = () => (
  <div>
    <p>Unfortunately, we haven&apos;t been able to load any tutorials.</p>
  </div>
);

export default TutorialLoadErrorMessage;
