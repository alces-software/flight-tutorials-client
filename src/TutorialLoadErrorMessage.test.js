// @flow
/*=============================================================================
 * Copyright (C) 2017 Stephen F. Norledge and Alces Flight Ltd.
 *
 * This file is part of Flight Tutorials.
 *
 * All rights reserved, see LICENSE.txt.
 *===========================================================================*/
import React from 'react';
import { render } from 'react-dom';
import renderer from 'react-test-renderer';

import TutorialLoadErrorMessage from './TutorialLoadErrorMessage';

it('renders without crashing', () => {
  const node = document.createElement('div');
  render(<TutorialLoadErrorMessage />, node);
});

it('renders correctly', () => {
  const tree = renderer.create(
    <TutorialLoadErrorMessage />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
