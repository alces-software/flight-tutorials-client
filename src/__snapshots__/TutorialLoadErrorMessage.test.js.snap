// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <p>
    Unfortunately, we haven't been able to load any tutorials.
  </p>
</div>
`;
